cibuildwheel/macos.py

```diff
@@ -229,7 +228,7 @@ def build(options: BuildOptions):
             call(['which', 'python'], env=virtualenv_env)
 
             if options.before_test:
-                before_test_prepared = prepare_command(options.before_test, project=abs_project_dir)
+                before_test_prepared = prepare_command(options.before_test, project='.', package=options.package_dir)
```

Comment:
`{package}` does seem a bit like it would be the name of the package rather than a directory? So why wouldn't this be `package_dir` (as we already call it in the arguments to cibuildwheel) ?
But we already had the same with `{project}` ofc, so maybe it's good to keep it consistent like this.

Reply:
good point... yes, the option is called `package_dir`... how would you feel about renaming the command line option to `package`?

Reply:
Hmmm, not too sure about that. I kind of like that it still refers to a directory, actually.

Reply:
Probably best to keep this consistent with `{project}` then, as you say.